docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+  ]
+
+  provider = liqo.rome
+
+  cluster_id      = liqo_generate.generate.cluster_id
+  cluster_name    = liqo_generate.generate.cluster_name
+  cluster_authurl = liqo_generate.generate.auth_ep
+  cluster_token   = liqo_generate.generate.local_token
+
+}
+```
+
+You can check the peering status by running:
+
+```bash
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that the cross-cluster network tunnel has been established, and an outgoing peering is currently active (i.e., the _Rome_ cluster can offload workloads to the _Milan_ one, but not vice versa):
+
+```text
+NAME    TYPE        OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+milan   OutOfBand   Established        None               Established   Established      12s
+```
+
+At the same time, you should see a virtual node (`liqo-milan`) in addition to your physical nodes:
+
+```bash
+kubectl get nodes
+```
+
+```text
+NAME                 STATUS   ROLES                  AGE     VERSION
+liqo-milan           Ready    agent                  14s     v1.23.6
+rome-control-plane   Ready    control-plane,master   7m56s   v1.23.6
+rome-worker          Ready    <none>                 7m25s   v1.23.6
+```
+
+### Liqo offload resource
+
+If you want to deploy an application that is scheduled onto Liqo virtual nodes, you should first create a namespace where your pod will be started.\
+This can be achieved with `kubernetes_namespace` of [kubernetes](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs) provider configured with config file or parameters of local cluster.
+
+Then tell Liqo to make this namespace eligible for the pod offloading.\
+The resource in charge of doing this is `liqo_offload` of the same [liqo](https://google.com) provider of `liqo_peering` resource:
+
+```
+resource "liqo_offload" "offload" {
+
+  depends_on = [
+    helm_release.install_liqo_rome,
+    kubernetes_namespace.namespace
+  ]
+
+  provider = liqo.rome
+
+  namespace = "liqo-demo"
+
+}
+```
+
+```{admonition} Note
+The virtual nodes have a [taint](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) that prevents the pods from being scheduled on them.
+The Liqo webhook will add the toleration for this taint to the pods created in the liqo-enabled namespaces.
+```
+
+Since no further configuration is provided, Liqo will add a suffix to the namespace name to make it unique on the remote cluster (see the dedicated [usage page](/usage/namespace-offloading.md) for additional information concerning namespace offloading configurations).
+
+### Deploy simple pod on offloaded namespace
+
+At this point the basic setup is completed and next resources created by Terraform will deploy a standard Kubernetes application in a multi-cluster environment as you would do in a single cluster scenario.
+
+The deployed resources are t two pods running an `nginx` image and a Service exposing the pods to the cluster.
```

Comment:
```suggestion
The deployed resources are two pods running an `nginx` image and a Service exposing the pods to the cluster.
```

Reply:
This section is removed following Marco Iorio's suggestions.